packages/gasket-fetch/lib/index.d.ts

```diff
@@ -1,4 +1,2 @@
-function fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
-export default fetch
-export { fetch }
-export { Headers, Request, Response, AbortController };
```

Comment:
This is blowing up typescript test here:
https://github.com/godaddy/gasket/blob/fetch-tune-types/packages/gasket-typescript-tests/test/fetch.spec.ts#L2

Since AbortController is in node LTS, I don't think the polyfill is beneficial and we can drop it in the next major release. I think it is safe to assume, these exported TS types can be dropped, and so can that TS test I linked.

Reply:
https://github.com/godaddy/gasket/pull/394/files#diff-573dd80318543dedf7e093f0cd401209b9edec8913b7101d549268a8b000331a